CalibTracker/SiStripChannelGain/python/SiStripGainsPCLWorker_cfi.py

```diff
@@ -1,7 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
-SiStripGainsPCLWorker = cms.EDAnalyzer(
-    "SiStripGainsPCLWorker",
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+SiStripGainsPCLWorker = DQMEDAnalyzer(
+    'SiStripGainsPCLWorker',
```

Comment:
I would suggest not to change the `"` to `'`, to reduce the number of changes.

Reply:
Artifact of the regexp used in my automated conversion. I can try to filter out these lines...